absl/base/attributes.h

```diff
@@ -123,8 +123,12 @@
 #define ABSL_ATTRIBUTE_NO_TAIL_CALL __attribute__((disable_tail_calls))
 #elif defined(__GNUC__) && !defined(__clang__)
 #define ABSL_HAVE_ATTRIBUTE_NO_TAIL_CALL 1
+#if defined(__e2k__)
+#define ABSL_ATTRIBUTE_NO_TAIL_CALL
+#else
 #define ABSL_ATTRIBUTE_NO_TAIL_CALL \
   __attribute__((optimize("no-optimize-sibling-calls")))
+#endif
```

Comment:
instead of this, please add `&& !defined(__e2k__)` on line 124

Reply:
changed

Reply:
Close:(Atributos.h)

Reply:
Close:(puertos e2k: e2k)